Native profiler v0.13.0 (#18883)
DataDog/documentation: content/en/profiler/enabling/ddprof.md

## Configuration

> 

Author: This table is being removed as it is redundant compared to the link with the versioned documentation we are providing.
The documentation is always up to date in `ddprof` github repository.

- `disable` disables the logs entirely.
- Any other value is treated as a file path, with a leading `/` designating an absolute path.

> ### Pidmode

Author: issue on pidmode should be resolved before we advertise this again 
https://github.com/DataDog/ddprof/issues/283

> If you want to instrument all running process, you can try out the `--global` option.
> Global mode is intended for debug purposes. This requires elevated permissions. Depending on your setup, this can mean running as root, granting `CAP_PERFMON`, `CAP_SYSADMIN`, or setting `perf_event_paranoid` to `-1`.

Reviewer: ```suggestion
To instrument all running process, use the `--global` option. Global mode is intended for debug purposes and requires elevated permissions. Depending on your setup, this can mean running as root, granting `CAP_PERFMON`, `CAP_SYSADMIN`, or setting `perf_event_paranoid` to `-1`.
```
